cmd/tools/vdoc/testdata/basic/basic.out

```diff
@@ -1,3 +1,7 @@
+Warning: `markdown` exists, but is not updated.
+V will continue, since updates can fail due to temporary network problems,
+and the existing module `markdown` may still work.
+--------------------------------------------------
```

Comment:
How did this get generated?

Reply:
Weird ... Perhaps there was a temporary network error while running `v install markdown`, or perhaps there was a local change in ~/.vmodules/markdown for some reason, and `v update markdown` failed (afaik it is run when doing `v doc`, when vdoc is not yet compiled (or after `v self`).

Imho it should disappear after doing `rm -rf ~/.vmodules/markdown`, followed by `v install markdown` again.

Imho that .out file should not be changed by this PR.